src/connector/src/source/nats/source/reader.rs

```diff
@@ -73,6 +73,9 @@ impl SplitReader for NatsSplitReader {
                     }
                 },
             },
+            // We have record on this Nats Split, contains the last seen offset (seq id) or reply subject
+            // We do not use the seq id as start position anymore,
+            // but just let the reader load from durable consumer on broker.
```

Comment:
I understand these comments, but you didn't make any change to the implementation code (L58 ~ L72). Why & how it works?

Reply:
We use api `get_or_create_consumer` when building stream consumer. [api ref](https://docs.rs/async-nats/0.37.0/async_nats/jetstream/stream/struct.Stream.html#method.get_or_create_consumer)
If we utilize an existing durable consumer, the provided config should be the same as the consumer created. So the config here should never change, always align to the one conducted from with clause. 

following changes in https://github.com/risingwavelabs/risingwave/pull/18895